plugins/modules/win_power_plan.py

```diff
@@ -12,14 +12,22 @@
   - This module will change the power plan of a Windows system to the defined string.
   - Windows defaults to C(balanced) which will cause CPU throttling. In some cases it can be preferable
     to change the mode to C(high performance) to increase CPU performance.
+  - One of I(name) or I(guid) must be provided.
 options:
   name:
     description:
       - String value that indicates the desired power plan.
       - The power plan must already be present on the system.
       - Commonly there will be options for C(balanced) and C(high performance).
     type: str
-    required: yes
+    required: false
+  guid:
+    description:
+      - String value that indicates the desired power plan.
```

Comment:
This should be worded a bit differently as it's essentially the same description as `name`.

Reply:
updated the doc and added change fragment